users/models.py

```diff
@@ -1,3 +1,11 @@
-# from django.db import models
+from django.db import models
+from django.contrib.auth.models import User
 
-# Create your models here.
+
+class Profile(models.Model):
+    user = models.OneToOneField(User, on_delete=models.CASCADE)
+    image = models.ImageField(default='default.jpg', upload_to='profile_pics')
+    dob = models.DateField()
+
+    def __str__(self):
+        return f'{self.user.username} Profile'
```

Comment:
please add a blank line at the end

Reply:
There is a blank line in line 12. 
Do you mean to add another one (line 13) ?

Reply:
no only line 12, but I don't see a blank line there :) https://github.com/beyond-io/picATrip/blob/df9933d0d7c3f7c1e5957314c765280e9dc9cb9d/users/models.py

Reply:
@Leeadar

Reply:
I'm sorry I didn't send my answer, do you see it now? :)

Reply:
yes but it's the notepad++ that added a fake blank file at the end, git doesn't do these things:)
also I opened it in vim to verify:
![image](https://user-images.githubusercontent.com/51245973/114352498-9730c700-9b74-11eb-8f30-0b1bdc785b0d.png)

Reply:
I tried to add a new line (and then delete line 13), is it work now ?

Reply:
no.. it shows no change in this file

Reply:
How can I open the file with vim like you did? I searched and didn't find how can I do that.
Maybe I can add the blank line in the vim editor... (I tried also with notepad, and it also shows me that line 12 is blank)

Reply:
It's a unix based text editor so you have to run on another OS.. but I think if you open it in any IDE (visual studio, pycharm.. etc) you won't see that empty line

Reply:
With Visual Studio code is the same result..
 
![image](https://user-images.githubusercontent.com/72739568/114379378-6ad77380-9b91-11eb-8208-287178d983ab.png)

Reply:
hmm so maybr you forgot to add/commit that file when you pushed to git? if you run `git status` do you see it?

Reply:
No.. its says 'nothing to commit, working tree clean' (in user-model branch)

Reply:
You could look inside vagrant, /usr/bin/vi
It look ok, for me

Reply:
Following my meet with Guy we decided to keep this file as before for now. Maybe there is a problem with Flake8 that indicate an extra new line